resources.md

```diff
@@ -205,10 +205,8 @@ inside a graphical user interface is often called a terminal window.
 - [alpha.wallhaven.cc](http://alpha.wallhaven.cc)
 - [simpledesktops.com](http://simpledesktops.com)
 - [subtlepatterns.com](http://subtlepatterns.com)
-- [squidfingers.com](http://www.squidfingers.com/patterns)
 - [unsplash.com](https://unsplash.com)
-- [nik.bot.nu](https://nik.bot.nu/browse.html)
-- [4walled.cc](http://4walled.cc)
```

Comment:
Can you make 4walled point to https://alpha.wallhaven.cc/ since that's what it redirects to?